app/models/neighborhood.rb

```diff
@@ -17,4 +17,16 @@ def editable_by?(user)
   def destroyable_by?(user)
     user.admin?
   end
+
+  def self.order_by_name(page)
+    order(:name).page(page).per(10)
```

Comment:
pagination is not a concern of the ordering. I'm not sure there's tremendous value to this class method